### DISCUSSION TITLE
Trigger form submit with `enhance` without clicking submit button

### DISCUSSION THREAD

**Discussion by DePasqualeOrg** [sveltejs/kit#11125] [Q&A] [answered]
I need to submit a form using `enhance` programmatically, without clicking the submit button. Calling `deleteProjectForm.submit()` bypasses the `enhance` function. Calling `deleteProjectButton.click()` doesn't submit the form. How can I trigger a form submit that uses the `enhance` function?

```svelte
<form use:enhance={handleDeleteProject} bind:this={deleteProjectForm}  method="POST" action="?/delete">
   <button bind:this={deleteProjectButton} type="submit">Delete</button>
</form>
```

**Comment by teemingc** [comment, marked as answer]
Try deleteProjectForm.requestSubmit()

**Reply by DePasqualeOrg** [reply]
That works. Thanks!
